sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -456,18 +456,18 @@ inline void zeParseError(ze_result_t ZeError, std::string &ErrorString) {
   } // switch
 }
 
-ze_result_t ZeCall::doCall(ze_result_t ZeResult, std::string ZeName,
-                           std::string ZeArgs, bool TraceError) {
-  zePrint("ZE ---> %s%s\n", ZeName.c_str(), ZeArgs.c_str());
+ze_result_t ZeCall::doCall(ze_result_t ZeResult, const char *ZeName,
+                           const char *ZeArgs, bool TraceError) {
+  zePrint("ZE ---> %s%s\n", ZeName, ZeArgs);
 
   if (ZeDebug & ZE_DEBUG_CALL_COUNT) {
     ++(*ZeCallCount)[ZeName];
   }
 
   if (ZeResult && TraceError) {
-    std::string ErrorString;
+    const char *ErrorString = "Unknown";
```

Comment:
Why did you need to do "Unknown" instead of "" before? BTW, I assume this value is always unused, i.e. `zeParseError` always updates the value, right?

Reply:
I chose unknown because there is a case where zeParseError has a default in the case, and doesn't set ErrorString.  I can change it to "", but I thought "Unknown" worked nicest.  I think it is unimportant, and in practice will be completely unused.

Reply:
OK